app/models/conversion_host.rb

```diff
@@ -56,6 +56,22 @@ def ipaddress(family = 'ipv4')
     resource.ipaddresses.detect { |ip| IPAddr.new(ip).send("#{family}?") }
   end
 
+  def get_cpu_limit
+    value = cpu_limit || Settings.transformation.limits.cpu_limit_per_host
+    value == 'unlimited' ? value : "#{value.to_i / active_tasks.size}"
+  end
+
+  def get_network_limit
+    value = network_limit || Setting.transformation.limits.network_limit_per_host
+    value == 'unlimited' ? value : "#{value.to_i / active_tasks.size}"
+  end
+
+  def apply_virtv2v_limits(path, limits)
```

Comment:
Are there any reasonable defaults we can set for `path` and/or `limits`?

Reply:
Nope. Path is stored in the task and specific to a single migration, so no default. And limits is built by the task object from different sources, so the default can be `{}`.